boards/shields/nrf7002eb2/doc/index.rst

```diff
@@ -0,0 +1,72 @@
+.. _nrf7002eb2:
+
+nRF7002 EB II
+#############
+
+Overview
+********
+
+The nRF7002 EB II is a versatile evaluation kit in the form of a thumbstick shield which connects to
+compatible Nordic host boards using the Nordic edge-connector.
+
+The nRF7002 EB II unlocks low-power Wi-Fi 6 capabilities for your host device. It supports dual-band Wi-Fi
+2.4GHz and 5GHz, and is based on the nRF7002 SoC. The shield also supports nRF7001 and nRF7000 SoCs
+through variant overlays.
+Seamlessly connect to Wi-Fi networks and leverage Wi-Fi-based locationing, enabling advanced
+features such as SSID sniffing of local Wi-Fi hubs.
+
+.. figure:: nrf7002eb2.jpg
+   :alt: nRF7002 EB II
+   :align: center
+
+   nRF7002 EB II
+
+Requirements
+************
+
+The nRF7002 EB II board is designed to fit straight into a Nordic edge-connector and uses SPI as the
```

Comment:
what is a "nordic edge connector"? It needs to be defined as a proper GPIO nexus similar to arduino uno, raspberry pi, etc and your shield should then be using this abstraction, and not manipulate `gpio0`, `gpio1` etc. directly
Or maybe this connector is `nordic-thingy53-edge-connector`? But in any case shield needs to be updated accordingly